cms/djangoapps/contentstore/views/component.py

```diff
@@ -353,9 +354,17 @@ def create_support_legend_dict():
                             )
                         )
 
+        #If using new problem editor, we select problem type inside the editor
+        # because of this, we only show one problem.
+        if category == 'problem' and use_new_problem_editor():
+            print('problemma',templates_for_category)
+            templates_for_category = [
+                template for template in templates_for_category if template['boilerplate_name'] == 'blank_common.yaml'
+                ]
```

Comment:
Where is the `blank_common.yaml` assignment coming from originally? Why not change code there? To avoid setting this flag in a number of places?

Reply:
ah great question. This method is the place where other checks are made upon the templates: are they enabled? are they supported? Do they work in this learning context? So I added this check here because it made sense.

Reply:
I see... makes sense to me!